plain-ai.mdx

```diff
@@ -2,16 +2,20 @@
 title: 'Plain AI'
 ---
 
-Quickly categorize and label new threads without lifting a finger. To turn on Plain’s AI features, go to **Settings → Workflow.\***
+Quickly categorize and label new threads without lifting a finger. To turn on Plain’s AI features, go to **Settings → Workflow.**
+
+All AI features in Plain are opt-in. More information on our use of OpenAI and our data processing policies can be found in our [DPA](https://www.plain.com/legal/dpa).
 
 <Frame>![Plain AI](/public/images/plain-ai.png)</Frame>
 
 **Auto-labelling**
 
 When a thread is created, Plain AI will add 1-2 labels automatically to a thread based on its content. If a thread was created with existing labels, no labels will be added. To best leverage Plain AI, make sure you’ve already set up [Labels](/labels) (**Settings → Labels**).
 
+**Thread titles**
+
+When enabled, this will automatically give each thread a human readable title based on the content of the support request. This is particularly useful in Slack and other channels which, unlike email, do not have a subject line you can rely on.
```

Comment:
'human readable' implies that previously titles weren't readable. I think 'short, descriptive' would be better here